cloud/defaults.go

```diff
@@ -39,6 +39,16 @@ const (
 	LatestVersion = "latest"
 )
 
+// GenerateInternalLBName generates a internal load balancer name, based on the cluster name.
+func GenerateInternalLBName(clusterName string) string {
```

Comment:
why is this needed? It should have been removed since #974

Reply:
must have missed something with the rebase, fixed it now